REQUIRE

```diff
@@ -1,6 +1,6 @@
-julia 0.5
-DataTables
-CategoricalArrays 0.0.6
-NullableArrays 0.0.10
+julia 0.6
+DataFrames 0.11
```

Comment:
There is no `DataFrames` `v0.11`.  How can this PR even be tested?

Reply:
You need to checkout DataFrames from master. If `Pkg` complains about the version, you can call `include("test/runtests.jl")` directly. I guess I should have kept 0.10 for now.

Reply:
Actually you need to checkout the master version of `DataFrames` (and, in my experience, it is difficult to do this unless you first create a clean package directory) then clone `StatsModels` directly into the package directory without going through Julia's package system then check out the branch then run `runtests.jl` directly.